lightning/src/ln/channel.rs

```diff
@@ -5434,7 +5434,8 @@ impl<'a, Signer: Sign, K: Deref> ReadableArgs<&'a K> for Channel<Signer>
 			(11, channel_type, option),
 		});
 
-		if channel_type.as_ref().unwrap().supports_unknown_bits() || channel_type.as_ref().unwrap().requires_unknown_bits() {
+		let chan_features = channel_type.as_ref().unwrap();
+		if chan_features.supports_unknown_bits() || chan_features.requires_unknown_bits() {
```

Comment:
nit: guess a simpler check would be `if chan_features != ChannelTypeFeatures::static_key_only()`

Reply:
Yea, but this "feels" more right - in the future we could/will add more bits and ideally we'd not have to change this code.